src/pytest_benchmark/plugin.py

```diff
@@ -1053,6 +1069,7 @@ def pytest_benchmark_generate_json(config, benchmarks, include_data):
                 "group": bench.group,
                 "name": bench.name,
                 "fullname": bench.fullname,
+                "params": bench.params,
```

Comment:
It might be useful, but I still think that the name "param" is misleading,
because it's not a single one but a combination of many. The correct name
would be "callspec id", as it is one of the parts which compose the pytest
"fullname"

Il 29/nov/2015 05:48 PM, "Ionel Cristian Mărieș" <notifications@github.com>
ha scritto:
>
> In src/pytest_benchmark/plugin.py:
>
> > @@ -1053,6 +1069,7 @@ def pytest_benchmark_generate_json(config,
benchmarks, include_data):
> >                  "group": bench.group,
> >                  "name": bench.name,
> >                  "fullname": bench.fullname,
> > +                "params": bench.params,
>
> Would it make sense to also have "param" here? ... a convenient mashup of
params (and otherwise hard to recreate because it depends on some ordering).
>
> —
> Reply to this email directly or view it on GitHub.